accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/RelationableSqlRender.java

```diff
@@ -105,6 +108,14 @@ protected RelationInfo render(Model baseModel)
                     getRelationableAlias(baseModel.getName()),
                     tableJoinCondition.apply(getRelationableAlias(baseModel.getName())));
         }
+        if (!calculatedRequiredRelationshipInfos.isEmpty()) {
+            for (CalculatedFieldRelationshipInfo calculatedFieldRelationshipInfo : calculatedRequiredRelationshipInfos) {
+                tableJoinsSql += format(" LEFT JOIN (%s) AS \"%s\" ON %s",
```

Comment:
Could you change `tableJoinSql` to a `StringBuilder`? There're string concat in a loop. Using `StringBuilder` is better.